phpstan.neon

```diff
@@ -15,6 +15,24 @@ parameters:
         -
             message: '#Symfony\\Component\\Config\\Definition\\Builder\\NodeParentInterface::scalarNode\(\)#'
             path: %currentWorkingDirectory%/packages/framework/src/Component/UploadedFile/Config/UploadedFileConfigDefinition.php
+        - '#(Method .+::.+\(\) should return .+ but returns PHPUnit\\Framework\\MockObject\\MockObject\.)#'
+        - '#(Property .+::\$.+ \(.+\) does not accept PHPUnit\\Framework\\MockObject\\MockObject\.)#'
+        -
+            message: '#Property Doctrine\\ORM\\Mapping\\ClassMetadataInfo::\$discriminatorColumn \(array\) does not accept null\.#'
+            path: %currentWorkingDirectory%/packages/framework/src/Component/EntityExtension/EntityExtensionParentMetadataCleanerEventSubscriber.php
+        -
+            message: '#Argument of an invalid type Symfony\\Component\\Validator\\Constraint supplied for foreach, only iterables are supported\.#'
+            path: %currentWorkingDirectory%/packages/framework/src/Form/JsFormValidatorFactory.php
+        - '#Method Doctrine\\Common\\Persistence\\ObjectManager::flush\(\) invoked with 1 parameter, 0 required\.#'
+        #TODO delete after deleting the deprecated method in the next major release#
```

Comment:
will someone know about this ? is it covered by Releaser?

Reply:
I'll create PRs for these 2 changes